src/libs/StatusBar/index.web.ts

```diff
@@ -0,0 +1,20 @@
+import StatusBar from './types';
+
+StatusBar.getBackgroundColor = () => {
+    const element = document.querySelector('meta[name=theme-color]');
+
+    if (element && 'content' in element && (typeof element.content === 'string' || typeof element.content === 'symbol')) {
+        return element.content;
+    }
+
+    return null;
+};
+
+StatusBar.setBackgroundColor = (backgroundColor) => {
+    const element = document.querySelector('meta[name=theme-color]');
+    if (element && 'content' in element) {
+        element.content = backgroundColor;
+    }
```

Comment:
```suggestion
    const element = document.querySelector<HTMLMetaElement>('meta[name=theme-color]');
    if (!element) {
        return;
    }
    element.content = backgroundColor as string;
```

We can simplify if using `HTMLMetaElement`.